### PR TITLE
remove camelCase

### PR DESCRIPTION
**Pull request by tidalf**

### CODE REVIEW

File: irbt/robot.py [hunk 1]
```diff
@@ -24,25 +24,25 @@ def __init__(self, robot):
             self.dock = functools.partial(robot._cmd, "dock")
             self.status = functools.partial(robot._cmd, "status")
 
-    def __init__(self, Cloud=None, id=None, outputRaw=None):
+    def __init__(self, cloud=None, id=None, output_raw=None):
         self.command = Robot.Commands(self)
         # if no cloud connexion is passed,create one using provided credentials
-        if not Cloud:
+        if not cloud:
```
Comment (showi) [inline irbt/robot.py]: raise here

File: irbt/robot.py [hunk 2]
```diff
@@ -24,25 +24,25 @@ def __init__(self, robot):
...
             print("You need to provide a cloud connection")
         else:
-            self._Cloud = Cloud
+            self._cloud = cloud
```
Comment (showi) [inline irbt/robot.py]: you sure you want `_cloud` not `cloud`

Comment (tidalf) [reply irbt/robot.py]: well yeah for now

File: cli.py [hunk 3]
```diff
@@ -56,66 +56,63 @@
...
     enable_mqtt_logging()
 # instance Robot
 username = os.environ.get("IRBT_LOGIN")
 password = os.environ.get("IRBT_PASSWORD")
 
 # cloud connexion
-Cloud = Cloud(username=username, password=password)
+cloud = Cloud(username=username, password=password)
+
+if not args.robot_id:
```
Comment (showi) [inline cli.py]: `robot_id = args.robot_id if args.robot_id else  list(cloud.robots())[0]`

Comment (showi) [reply cli.py]: so robot_id is seen as defined and not pull from nowhere

File: irbt/robot.py (hunk 1 above)
Comment (showi) [reply irbt/robot.py]:
```
if not cloud:
  raise Exception('NoCloudParameter')
self._cloud = cloud
```

Comment (tidalf) [reply irbt/robot.py]: done

File: cli.py (hunk 3 above)
Comment (tidalf) [reply cli.py]: done

**Pull request merged**